src/app/beer_garden/__main__.py

```diff
@@ -26,6 +27,12 @@ def main():
     # Absolute first thing to do is load the config
     beer_garden.load_config(sys.argv[1:])
 
+    # Register the events handler before the application starts
+    #beer_garden.logger.info("Starting local events manager...")
+    #beer_garden.establish_events_manager()
+
+    beer_garden.establish_events_queue()
+
```

Comment:
Let's move this out of this file - anything in `__main__` is really difficult to test, and I don't think there's a reason why we can't create the queue as part of the application startup.

Reply:
I can work on moving this over to the app.py code.